setup.py

```diff
@@ -50,6 +50,7 @@
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
```

Comment:
Thanks! Might as well include Python 3.11 now too.

https://www.python.org/downloads/release/python-3110/

Reply:
Thanks for the heads-up! Done in 601dc55.